tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -152,6 +178,9 @@ function delete_runtimeclasses() {
 			shim_name+="-${MULTI_INSTALL_SUFFIX}"
 			sed -i -e "s|${canonical_shim_name}|${shim_name}|g" /opt/kata-artifacts/runtimeclasses/${canonical_shim_name}.yaml
 		fi
+		
+		adjust_shim_for_nfd "${shim}" "${expand_runtime_classes_for_nfd}"
```

Comment:
Hi @fidencio , why we need to adjust shim for NFD if the runtimeclass is gonna be deleted anyway?